exchanges/kucoin/kucoin_websocket.go

```diff
@@ -6,9 +6,11 @@ import (
 	"errors"
```

Comment:
```
[ERROR] | WEBSOCKET | 29/07/2024 13:32:51 | Kucoin websocket: subscription failure, ticker all : subscription template containing PairSeparator with must contain either specific Asset or AssetSeparator, orderbook all : subscription template containing PairSeparator with must contain either specific Asset or AssetSeparator
```
I have the following pairs enabled:
```json
 "spot": {
      "assetEnabled": true,
      "enabled": "BTC-USDT,AVA-USDT,FET-BTC,FET-ETH,ANKR-BTC,XMR-BTC,XMR-ETH,MTV-BTC,MTV-ETH",
```

Reply:
@gloriousCode Yep, I could reproduce this before my recent backported fix, and it's gone with my latest fix.

Fixed 4782a10